tsdb/engine/tsm1/cache.go

```diff
@@ -231,10 +231,10 @@ func (c *Cache) WriteMulti(values map[string][]Value) error {
 	}
 	c.mu.RUnlock()
 
+	c.mu.Lock()
```

Comment:
@oiooj `c.entry(k)` contains a low-contention path that only requires an `RLock`, yet your change will always mean a `Lock` is needed. Do you have some data to support this change being more performant than what already exists?

Could you provide some benchmarks or other data that shows this improves performance?